cmd/node/node.go

```diff
@@ -150,8 +157,8 @@ type SpacemeshApp struct {
 	Config            *cfg.Config
 	grpcAPIService    *api.SpacemeshGrpcService
 	jsonAPIService    *api.JSONHTTPServer
-	newgrpcAPIService *grpcserver.Server
-	newjsonAPIService *grpcserver.JSONHTTPServer
+	newgrpcAPIService *server.Server
```

Comment:
It's not as immediately obvious from the package name `server` what this package does, `grpcserver` is a bit clearer and more intuitive isn't it?

Reply:
Ok, I will change this into grpcserver.Server